### PR TITLE
chore(deps): update source-map

### PR DESCRIPTION
**Pull request by sibiraj-s**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

[Sourcemap v0.7] has some performance improvements.
Also removed an unused dependency `del`

### Breaking Changes

None

### Additional Info

None

[Sourcemap v0.7]: https://github.com/mozilla/source-map/blob/master/CHANGELOG.md#070

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Sorry, we can't accept this PR, because webpack and other modules uses old sync version of `source-map`, also it is used only for errors, so no big perf here

File: src/index.js [hunk 1]
```diff
@@ -487,45 +491,41 @@ class TerserPlugin {
...
-      if (worker) {
-        return worker.end();
-      }
-
-      return Promise.resolve();
-    });
+    await Promise.all(scheduledTasks);
+    if (worker) {
+      await worker.end();
+    }
```
Comment (evilebottnawi) [inline src/index.js:528]: But feel free to send a PR with refactor on `await`/`async`

Comment (sibiraj-sr) [reply src/index.js:528]: can i just push the changes to the same pr and remove the source-map changes?

**Comment by sibiraj-sr** [comment]
> because webpack and other modules uses old sync version of source-map

I thought of that, but since there is nothing breaking, I made this PR. I thought may be other packages could update it one at a time.

> also it is used only for errors, so no big perf here

I am not completely aware of the internals, We have quite a large app, and minify phase is quite slow, just felt like any improvement is good to have 😅 .

File: src/index.js (hunk 1 above)
Comment (evilebottnawi) [reply src/index.js:528]: Let's do it in separate PR

**Comment by evilebottnawi** [comment]
> I thought of that, but since there is nothing breaking, I made this PR. I thought may be other packages could update it one at a time.

We want to do it together with all future codebase - future

> I am not completely aware of the internals. We have quite a large app, and minify phase is quite slow, just felt like any improvement is good to have sweat_smile .

Updating source map do not improve perf in this case :smile:

**Pull request closed without merging**